website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,176 @@
+These can include basic Kubernetes features or features that are specific to the cloud provider (such as AWS).
+
+Layered *constraints* are applied when a pod makes requests for compute resources that cannot be met by current capacity.
+A pod can request *affinity* (to run in a particular zone or instance type) or a topology spread (to cause a set of pods be run across multiple nodes).
+The pod can ask to run only on nodes with a particular label or that have a certain amount of available memory or particular architecture type.
+
+The Kubernetes scheduler tries to match those constraints with available nodes.
+If the pod is unschedulable, facilities like the Cluster Autoscaler or Karpenter come in and try to create compute resources to match each pod's needs.
+When Karpenter tries to provision a node that can schedule an unschedulable pode, it compares its own constraints when choosing the node to create.
+
+As long as the requests are not outside of the provisioner's constraints, 
+Karpenter will look to best match the request, comparing mostly the same well-known labels and affinity features that the pod uses.
+Note that if the constraints are such that a match is not possible, the pod will not deploy.
+
+So, what constraints can you use as an application developer deploying pods that could be managed by Karpenter?
+
+* **Kubernetes**: Kubernetes features that Karpenters supports for scheduling nodes include node affinity based on [persistant volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#node-affinity) and [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector).
+It also supports [PodDisruptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) and [topologySpreadConstraints](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
+
+  From the Kubernetes [Well-Known Labels, Annotations and Taints](https://kubernetes.io/docs/reference/labels-annotations-taints/) page,
+you can see a full list of Kubernetes labels, annotations and taints that determine scheduling.
+Only a small set of them are implemented in Karpenter, including:
+
+  * **kubernetes.io/arch**: For example, kubernetes.io/arch=amd64
+  * **kubernetes.io/os**: For example, kubernetes.io/os=linux
+  * **node.kubernetes.io/instance-type**: For example, node.kubernetes.io/instance-type=m3.medium
+  * **topology.kubernetes.io/zone**: For example, topology.kubernetes.io/zone=us-east-1c
+
+
+* **AWS-specific**: Capacity types (node.k8s.aws/capacity-type) that are specific to AWS in Karpenter include spot and on-demand.
+Karpenter also supports AWS-specific accelerators, including:
+
+  * **nvidia.com/gpu**: For Nvidia graphics processing units
+  * **amd.com/gpu**: For AMD graphics processing units
+  * **aws.amazon.com/neuron**: For AWS Neuron machine learning applications
```

Comment:
None of these are aws-specific anymore.
```suggestion
* **AWS-specific**: Capacity types (node.k8s.aws/capacity-type) that are specific to AWS in Karpenter include spot and on-demand.
Karpenter also supports AWS-specific accelerators, including:

  * **nvidia.com/gpu**: For Nvidia graphics processing units
  * **amd.com/gpu**: For AMD graphics processing units
  * **aws.amazon.com/neuron**: For AWS Neuron machine learning applications
```

Reply:
Okay. Since all the AWS-specific settings are going, I'm going to remove the whole AWS-specific section and remove the bullet from the Kubernetes section.